boards/DCM/Test/Src/torquevectoring/Test_PID.cpp

```diff
@@ -0,0 +1,69 @@
+
+
+#include "Test_Dcm.h"
+
+extern "C"
+{
+    #include "torquevectoring/App_PID.h"
+    #include "torquevectoring/App_TorqueVectoring.h"
+}
+
+class PIDTest : public testing::Test
+{
+    protected:
+        void SetUp() override
+        {
+            return;
+        }
+        void TearDown() override
+        {
+            return;
+        }
+};
```

Comment:
You can just delete this class if its not used
I believe the class is only used if you use the TEST_F macro vs. the TEST macro

Reply:
@1mackenziekyle